<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Landscape</class>
 <widget class="QDialog" name="Landscape">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1038</width>
    <height>508</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Landscape</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="lbl_frame">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>781</width>
     <height>441</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_menu">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>470</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::PreventContextMenu</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(130, 164, 79);
border: none;
color: white;
</string>
   </property>
   <property name="text">
    <string>BACK</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_cost">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>470</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Total Cost:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_totalcost">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>470</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_items">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>60</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>ITEMS</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QFrame" name="frame_items">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>90</y>
     <width>181</width>
     <height>381</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="btn_shapes">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(94, 170, 142);
border: none;
color: white;</string>
    </property>
    <property name="text">
     <string>Shapes</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_icons">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(94, 170, 142);
border: none;
color: white;</string>
    </property>
    <property name="text">
     <string>Icons</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_shapes">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>90</y>
     <width>181</width>
     <height>371</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="btn_shapeback">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(130, 164, 79);
border: none;
color: white;
</string>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_shape">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(206, 185, 160);
border:none;</string>
    </property>
    <item>
     <property name="text">
      <string>Circle</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Parallelogram</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Rectangle</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Square</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Triangle</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_icons">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>90</y>
     <width>181</width>
     <height>381</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="btn_iconback">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(130, 164, 79);
border: none;
color: white;
</string>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_icon">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(206, 185, 160);
border:none;</string>
    </property>
    <item>
     <property name="text">
      <string>Flower</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Ground Covers</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Tree</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QLabel" name="lbl_cost_3">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>470</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Baht</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_bg">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1041</width>
     <height>521</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>lbl_bg</zorder>
  <zorder>lbl_frame</zorder>
  <zorder>btn_menu</zorder>
  <zorder>lbl_cost</zorder>
  <zorder>lbl_totalcost</zorder>
  <zorder>lbl_items</zorder>
  <zorder>frame_items</zorder>
  <zorder>frame_shapes</zorder>
  <zorder>frame_icons</zorder>
  <zorder>lbl_cost_3</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
